ui/eslint.config.mjs

```diff
@@ -8,7 +9,8 @@ const storyBookConfig = {
     storybook,
   },
 };
-
+// eslint-plugin-tailwindcss does not support the latest tailwindcss version.
+// Therefore, we use eslint-plugin-better-tailwindcss instead, which is being regularly updated and a work in progress.
```

Comment:
This kinda detail isn't necessary. The original plugin is irrelevant for us and the repo now.